test/integration/organizations/staff/adoption_application_reviews_test.rb

```diff
@@ -74,6 +126,15 @@ class Organizations::Staff::AdoptionApplicationReviewsTest < ActionDispatch::Int
         follow_redirect!
         assert_equal "Unauthorized action.", flash[:alert]
       end
+
+      should_eventually "not be able to view application details" do
```

Comment:
What is this testing?

Reply:
I tried to replicate the way it was done for index page but it was not actually working as expected so removed it. Infact existing test for index also uses should_eventually which is not actually passing but just skipping it. I think we can removed deactivated staff block but have kept old code as it is for now.